components/eam/src/physics/cam/gw_drag.F90

```diff
@@ -118,6 +118,11 @@ module gw_drag
   ! namelist 
   logical          :: history_amwg                   ! output the variables used by the AMWG diag package
 
+  logical  :: use_gw_convect_old          ! switch to enable legacy behavior
+  real(r8) :: gw_convect_plev_src_wind    ! reference pressure level for source wind for convective GWD
+  real(r8) :: gw_convect_hdepth_min       ! minimum hdepth for for convective GWD spectrum lookup table
+  real(r8) :: gw_convect_storm_speed_min  ! minimum convective storm speed for convective GWD
```

Comment:
Add expected units for gw_convect_storm_speed_min.

Reply:
fixed